styles.css

```diff
@@ -21,11 +21,39 @@ section > div {
   max-width: 1440px;
 }
 
+/* ---------------------popUp section----------------------- */
+
 a {
   margin: 0;
   padding: 0;
+  text-decoration: none;
+  color: white;
+}
+
+.popup-para {
+  align-content: center;
+  margin: 0 16px;
+  margin-left: 18px;
+}
+
+.popup-photo {
+  margin: 14px 16px;
+  align-content: center;
 }
 
+#top-popup {
+  margin: 14px 16px;
+  align-content: center;
+}
+
+.btn {
+  display: flex;
+  justify-content: space-around;
+  margin: 14px 16px;
+}
+
```

Comment:
- [ ] **[OPTIONAL]** Kindly, make sure the popup modals are responsive and match the figma design. This is a **project requirement** missed by the previous reviewers. Implementing it will be essential for your learning journey, so please consider implementing it.

  - Make sure the popup match the figma design for desktop version and mobile version

**Your project popup on desktop version**
![image](https://user-images.githubusercontent.com/74411135/233800170-f179a719-1c50-4c11-afba-e96c9605ae08.png)

**Your project popup on mobile version**
![image](https://user-images.githubusercontent.com/74411135/233800264-ae1c5d9f-8474-43a0-87c9-74d5573b6170.png)

**figma design on desktop version**
![image](https://user-images.githubusercontent.com/74411135/233800205-4282f989-d4f0-4fe3-a504-4033240bfa57.png)

**figma design on mobile version**
![image](https://user-images.githubusercontent.com/74411135/233800218-d97eb3b1-0e21-40d8-86a4-037ede8eaa94.png)
 
This will make your project prettier and make the user know you're an awesome developer as you can precisely follow a professional web developer's design 😃 💪